kernel/kernel-api/src/main/java/io/delta/kernel/internal/TransactionBuilderImpl.java

```diff
@@ -163,6 +164,9 @@ public Transaction build(Engine engine) {
     }
 
     // TODO In the future update metadata with new schema if provided
+    if (schema.isPresent() && !isNewTable) {
```

Comment:
I see 3 `schema.isPresent() && !isNewTable` calls in this class -- would it be cleaner and more expressive to assign this to a helper method? `isSetSchemaOp -> boolean`

Reply:
I shrunk it down to 2 :) but possibly? I'd probably get this for a follow on if that's OK.